package.json

```diff
@@ -79,8 +80,16 @@
     "typedoc": "0.19.2",
     "typedoc-plugin-markdown": "3.1.1",
     "typescript": "^4.2.2",
+    "vue": "^3.0.7",
```

Comment:
maybe "vue3": "npm:vue@^3.0.7", ?

Reply:
As you wish. I would let the current vue version the default one (default alias = vue).